src/pretix/api/views/order.py

```diff
@@ -219,27 +213,32 @@ def get_queryset(self):
         if 'invoice_address' not in self.request.GET.getlist('exclude'):
             qs = qs.select_related('invoice_address')
 
-        qs = qs.prefetch_related(self._positions_prefetch(self.request))
+        qs = qs.prefetch_related(*self._positions_prefetch(self.request))
```

Comment:
AFAICT currently there will allways only be one `Prefetch` returned from `self._positions_prefetch`. Is the list something you added for future updates or something you added, because you thought you need it and turned out you didn’t? I guess that is also where the new vars `items_base` and `subevents_base` come from – they are only used once. It does not hurt, I am just curious.

Reply:
Yup, these were part of my efforts to support ?pdf_data=true on the new endpoints, which… failed, and I did not roll them back fully, apparently. Will remove from the diff.